deployment/kubernetes/crop-health-api.yaml

```diff
@@ -85,20 +82,11 @@ spec:
     - websecure
   routes:
   - kind: Rule
-    match: Path(`/crop-health/redoc`) || Path(`/crop-health/openapi.json`) || Path(`/crop-health/docs`)
+    match: Path(`/crop-health/redoc`) || Path(`/crop-health/openapi.json`) || Path(`/crop-health/docs`) || PathPrefix(`/crop-health`) && !PathPrefix(`/crop-health/metrics`)
```

Comment:
Is this necessary to keep still? Aren't all requests going through fastapi, so this can be simplified?